sklearn/ridge.py

```diff
@@ -14,37 +15,47 @@
                     help="Don't fit intercept (assume data already centered)")
 parser.add_argument('--solver', default='auto',
                     help='Solver used for training')
-params = parse_args(parser, size=(1000000, 50), dtypes=('f8', 'f4'),
-                    loop_types=('fit', 'predict'))
+parser.add_argument('--alpha', type=float, default=1.0,
+                    help='Regularization strength')
+params = parse_args(parser, size=(1000000, 50), loop_types=('fit', 'predict'))
 
-# Generate random data
-X = np.random.rand(*params.shape).astype(params.dtype)
-Xp = np.random.rand(*params.shape).astype(params.dtype)
-y = np.random.rand(*params.shape).astype(params.dtype)
+# Load data
+X_train, X_test, y_train, y_test = load_data(
+    params, generated_data=['X_train', 'y_train'])
 
 # Create our regression object
-regr = Ridge(fit_intercept=params.fit_intercept,
+regr = Ridge(fit_intercept=params.fit_intercept, alpha=params.alpha,
              solver=params.solver)
 
 columns = ('batch', 'arch', 'prefix', 'function', 'threads', 'dtype', 'size',
            'time')
 
-print_header(columns, params)
-
 # Time fit
-fit_time, _ = time_mean_min(regr.fit, X, y,
+fit_time, _ = time_mean_min(regr.fit, X_train, y_train,
                             outer_loops=params.fit_outer_loops,
                             inner_loops=params.fit_inner_loops,
                             goal_outer_loops=params.fit_goal,
                             time_limit=params.fit_time_limit,
                             verbose=params.verbose)
-print_row(columns, params, function='Ridge.fit', time=fit_time)
 
 # Time predict
-predict_time, yp = time_mean_min(regr.predict, Xp,
+predict_time, yp = time_mean_min(regr.predict, X_test,
                                  outer_loops=params.predict_outer_loops,
                                  inner_loops=params.predict_inner_loops,
                                  goal_outer_loops=params.predict_goal,
                                  time_limit=params.predict_time_limit,
                                  verbose=params.verbose)
-print_row(columns, params, function='Ridge.predict', time=predict_time)
+
+if params.output_format == 'json':
```

Comment:
Why do we only report accuracies in JSON outputs?

Reply:
Added accuracy calculation for CSV output format, you may correct CSV printing to output accuracies if needed. I didn't that because of backward-compatibility

Reply:
There is no requirement for 1:1 compatibility of CSV formats here. My main requirement is that we should not entirely replace CSV outputs with JSON. If we can write less code by changing CSV outputs slightly, then I think it's a good idea. But not a problem if you need this merged soon.

Reply:
I removed `if params.output_format == 'json'`